docs/core/porting/versioning-sdk-msbuild-vs.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Versioning relationship between the .NET SDK, MSBuild and VS
```

Comment:
I wonder if this article would be better placed as a sibling to https://docs.microsoft.com/en-us/dotnet/standard/frameworks in the toc?

The title though, is too big.

```suggestion
title: .NET SDK, MSBuild, and Visual Studio versioning
```

Reply:
The information on this page is useful from a couple different angles, I don't particularly care where it lives, under "General Information --> About .NET" I'm planning to add a number of different topics that will link off to details located in other places, that would be fine in this case as well.
I'll leave it for @marcpopMSFT to decide if it should go somewhere else.